<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>290</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;lol&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="outputsList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>9</y>
      <width>311</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="outputsTab">
     <attribute name="title">
      <string>Outputs</string>
     </attribute>
     <widget class="QLabel" name="outputNameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="outputNameText">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="outputPinLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pin:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="outputPinText">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
     </widget>
     <widget class="QListView" name="outputTriggerListBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="outputTriggerLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Trigger:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addTriggerButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addOutputButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="inputsTab">
     <attribute name="title">
      <string>Inputs</string>
     </attribute>
     <widget class="QLabel" name="inputNameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addInputButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="inputPinText">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLabel" name="inputPinLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pin:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputNameText">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListView" name="inputsList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLabel" name="inputsLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Inputs</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputsLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Outputs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editOutputButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editInputButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>118</width>
      <height>231</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>line</zorder>
   <zorder>outputsList</zorder>
   <zorder>tabWidget</zorder>
   <zorder>inputsList</zorder>
   <zorder>inputsLabel</zorder>
   <zorder>outputsLabel</zorder>
   <zorder>editOutputButton</zorder>
   <zorder>editInputButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuoh">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuoh"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
